src/Farmer/Builders/Builders.RoleAssignment.fs

```diff
@@ -0,0 +1,20 @@
+[<AutoOpen>]
+module Farmer.Builders.RoleAssignment
+
+open Farmer
+open Farmer.Arm.RoleAssignment
+
+/// Creates a role assignment for a specific scope (a resource or subscription).
+let inline scoped_role_assignment name role (assignee:^t) scope =
```

Comment:
For standard functions, I would like to avoid `underscore_naming` - let's stick to standard `camelCasing`.